tests/Resources/ti.media.videoplayer.test.js

```diff
@@ -478,4 +478,27 @@ describe.androidARM64Broken('Titanium.Media.VideoPlayer', () => {
 		player.addEventListener('playing', () => finish());
 		win.open();
 	});
+
+	it.ios('App should not crash when setting url after video player creation (TIMOB-28217)', function (finish) {
+		this.timeout(10000);
+
+		win = Ti.UI.createWindow();
+		player = Ti.Media.createVideoPlayer({
+			top: 120,
+			autoplay: false,
+			backgroundColor: 'blue',
+			height: 300,
+			width: 300,
+			mediaControlStyle: Titanium.Media.VIDEO_CONTROL_DEFAULT,
+			scalingMode: Titanium.Media.VIDEO_SCALING_ASPECT_FIT,
+			showsControls: true,
+		});
+		player.url = '/movie.mp4';
+		win.add(player);
+		win.open();
+
+		win.addEventListener('open', () => {
```

Comment:
Feels like you should set the listener before calling open

Reply:
Done.